CMakeLists.txt

```diff
@@ -167,6 +169,33 @@ elseif()
   message(FATAL_ERROR "Cannot locate edbee-lib editor widget submodule source code, build abandoned!")
 endif()
 
+file(GLOB_RECURSE lua_files RELATIVE "${CMAKE_HOME_DIRECTORY}/src/mudlet-lua/lua/" "${CMAKE_HOME_DIRECTORY}/src/mudlet-lua/lua/*.lua")
+list(LENGTH lua_files lua_file_count)
+if(lua_file_count EQUAL 26)
+  message(STATUS "Found Mudlet & Geyser lua files")
+else()
+  message(WARNING "Found ${lua_file_count} Mudlet & Geyser lua files but 26 were expected:")
```

Comment:
What is the idea behind including this?

Reply:
I had _fun_ getting the files included in the CMake "make install" and I wanted to check that I had got the right number of files - if this changes in the future it will warn me to recheck that everything is still working...

Reply:
It just means it's harder to update the submodule because we can't just do a git pull, all for the five people who'll run make install... OK I guess but the benefit is hardly worth it.

Reply:
It will still work - it will just emit a warning if the count is off (it is a `WARNING` not a show-stopping `FATAL_ERROR`)

Reply:
Alright.